### PR TITLE
Add intern/senior ML job descriptions

### PR DESCRIPTION
**Pull request by tsolakoua**
@vmarkovtsev, we are in the process of reviewing and updating the job descriptions on Lever. 

The parts of Intro, Culture and Perks have already been reviewed and it’s recommended to be same as every other role for consistency. 

The “TEAM” and “ROLE” are the ones that you are responsible to update in case you think it’s necessary. 

In the other job descriptions, the "TEAM" describes the teams at source{d} and a bit of the specific one. An example is here: https://github.com/src-d/guide/pull/108 .

Feel free to make any changes you think is necessary and let me know if there is anything I can do from my side to help. Later I will apply the changes on Lever.

### CODE REVIEW

File: talent/job-descriptions/intern-ds.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+- At source{d} we are building the stack for the next generation of Machine Learning powered developer tools.
+
+- We have raised over eight million USD so far, and we are currently growing our team.
+
+- Our current monetization strategy is based on deploying our stack and tooling within large tech companies.
+
+## TEAM
+
+- The Machine Learning team is focused on providing intelligent insights into nearly all the world's open source code. This includes typical big data analysis as well as solving sophisticated Machine Learning problems. The former is conducted on Apache Spark clusters with up to 1,000 nodes, the latter leverage extensive GPGPU acceleration on the custom hardware and Deep Learning techniques, standing on top of source{d}'s original open source projects and Tensorflow. Often, arising ML problems are unsupervised and require much research in the unusual domain of ML on source code (MLoSC).
```
Comment (vmarkovtsev) [inline talent/job-descriptions/intern-ds.md]: the latter leverages

File: talent/job-descriptions/intern-ds.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+- source{d} tries to be as open in it's ongoing research as possible. If a handy developed tool can be open-sourced then it has to be. If a solved interesting problem can be disclosed, it has to be published in source{d}'s blog. If a funny dataset is compiled, it is published on data.world. Writing papers is welcomed. We talk on conferences literally every month.
+
+- To this moment, source{d}'s engineers have developed:
+- src-d/awesome-machine-learning-on-source-code - everything that we know about MLoSC.
```
Comment (vmarkovtsev) [inline talent/job-descriptions/intern-ds.md]: There are no links

File: talent/job-descriptions/intern-ds.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+- kmcuda - lightning fast K-means and K-nearest neighbours on NVIDIA GPUs. It allows us to cluster 3 million samples, each 256 dimensions, into 1000 clusters in less than 20 minutes with the outstanding precision using two NVIDIA Titan X cards.
+- minhashcuda - lightning fast Weighted MinHash on NVIDIA GPUs. This tool allowed us to scan all +17M git repositories in 30 minutes and find 1.5M duplicates (the results were
+published on data.world.
+- lapjv - Jonker-Volgenant algorithm to solve linear sum assignment problems, accelerated with Intel AVX2 instruction set. It is capable of transforming 4096 t-SNE dots into 64x64 images in 2 minutes.
+- wmd-relax - optimized Word Mover's Distance
+- hercules - Git repositories line burn down analysis command line tool built on top of source{d}'s go-git - Git client and server implementation in pure Go language.
+• sparkpickle - the tool to read PySpar RDD files without having to install Spark.
+
+### The following notable technical posts, papers and talks exist:
+- Source Code Identifier Embeddings
```
Comment (vmarkovtsev) [inline talent/job-descriptions/intern-ds.md]: There are no links

File: talent/job-descriptions/intern-ds.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+- Analyzing GitHub, How Developers Change Programming Languages Over Time
+- GitHub Contributions Graph: Analyzing Pagerank & Proving the 6 Handshakes Theory
+- Similarity of GitHub Repositories by Source Code Identifiers
+- 397 Languages, 18,000,000 GitHub repositories, 1.2 billion files, 20 terabytes of code: Spaces or Tabs
+- Topic modeling of public repositories at scale using names in source code - paper
+- Hands on with the most starred GitHub repositories
+- Source code abstracts classification using CNN
+
+### The following datasets published:
+- Names in source code extracted from 13 000 000 GitHub repositories. Not people! 30GB
```
Comment (vmarkovtsev) [inline talent/job-descriptions/intern-ds.md]: There are no links

File: talent/job-descriptions/intern-ds.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+- October 2016 GitHub repositories not marked as forks but very similar to each other
+- Readme files found in all GitHub repositories (16M, October 2016
+- ≈ 452,000,000 commits' metadata taken from 16,000,000 repositories on GitHub (Oct 2016)
+
+
+ROLE
+
+- Strong computer science and machine learning background is essential for data science team members. You will be expected to be a passionate, skilful engineer who is able to produce amazing results quickly and reliably. Coding skills are important; we are using Python 3 in our research and production prototyping. Besides, we occasionally code in Go, C++ and CUDA.
+
```
Comment (vmarkovtsev) [inline talent/job-descriptions/intern-ds.md]: MLoSC shares common ideas with Natural Language Processing, so solid **NLP** knowledge is also required. **Deep Learning** experience is highly appreciated.

**Comment by tsolakoua** [comment]
@vmarkovtsev All the links exist on Lever. It's a pain to add them here manually with Markdown one by one. I'll do the change you requested here and adapt them to Lever and AngelList.

**Comment by tsolakoua** [comment]
@vmarkovtsev Links added.

**Comment by eiso** [comment]
@vmarkovtsev can this be merged?

**Comment by tsolakoua** [comment]
@vmarkovtsev  @eiso Salary updated here as well as Lever/careers page.

File: talent/job-descriptions/senior-ds.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+## PERKS 
+
+ - EU Visa support.
+ - Open Source Days, 10% of the time you can work on any OSS project you choose.
+ - Flexible hours and the possibility of remote work.
+ - We go to conferences and other developer events ! ;)
+ - Free books. Company will buy any books that can help you be even better at your work.
+ - Spacious physical office, close to public transit and to the Retiro park.
+ - Kitchen, full of supplies from beers to fresh fruits.
+ - Monthly get-togethers, annual summer and winter Xmas parties, a retreat.
```
Comment (vmarkovtsev) [inline talent/job-descriptions/senior-ds.md]: I would change "a retreat" to "hackathons" because few people outside of the company know about it means

**Comment by vmarkovtsev** [comment]
@eiso ready

**Comment by eiso** [comment]
Merging this **but** the structure of the file will have to be updated in https://github.com/src-d/guide/pull/132

**Pull request merged**